app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorView.kt

```diff
@@ -77,7 +78,7 @@ class SyncIndicatorView(context: Context, attrs: AttributeSet) : LinearLayout(co
         syncStatusTextView.text = if (durationToMinsAgo == 0) {
           context.getString(R.string.syncindicator_status_synced_just_now)
         } else {
-          context.getString(R.string.syncindicator_status_synced, durationToMinsAgo)
+          context.getString(R.string.syncindicator_status_synced, "$durationToMinsAgo")
```

Comment:
Unnecessary template?

Reply:
As discussed, I am not making this change.